src/pages/Admin/AdminVenueRoomDetails.tsx

```diff
@@ -122,13 +122,28 @@ export const AdminVenueRoomDetails = ({
                       name={"toggle-" + index}
                       checked={room.isEnabled}
                       onClick={() => {
-                        updateRoom(!room.isEnabled);
+                        updateRoom(!room.isEnabled, "isEnabled");
                       }}
                     />
                     <span className="slider round"></span>
                   </label>
                   <div>Turn room {room.isEnabled ? "Off" : "On"}</div>
                 </div>
+                <div className="toggle-room">
+                  <label id={"toggle2-" + index} className="switch">
+                    <input
+                      type="checkbox"
+                      id={"toggle2-" + index}
+                      name={"toggle2-" + index}
+                      checked={room.isLabeled}
+                      onClick={() => {
+                        updateRoom(!room.isLabeled, "isLabeled");
+                      }}
```

Comment:
Inline function handlers like this will cause re-renders. They should be wrapped with `useCallback` outside of the JSX, then that passed here.

Reply:
Decided to call this OOS as this issue is all over the place in admin v1, and it's not worth cleaning up since we plan to remove it as soon as admin v3 reaches feature parity.